### PR TITLE
docs: document v1 pod aliasing issue

### PR DESCRIPTION
**Pull request by ffromani**
Document the known issues of the v1 algo.

Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +13,22 @@ Note this package will *NOT* restrict itself to use only cryptographically secur
...
+recreated with non-identical pod specs (e.g. changing pod resources requests/limits), the fingerprint will clash. So, two different set of pods will yield
+the same fingerprint. [Future versions of the fingerprint will address this issue](https://github.com/k8stopologyawareschedwg/podfingerprint/issues/3).
+
+#### more details
+
+The v1 fingerprint algorithm uses the `namespace+name` pair to identify a pod because of a conscious design decision due to a survey of the data source available
+to the perspective consumers of this package.
+Considering that the source of truth for resource allocation is the kubelet, and preferring node-local APIs, the agents running on the node have only few
+data source options:
+- the `pods` kubelet endpoint - to learn about resource allocation, but does not report the pod UID, only the `namespace+name` pair.
```
Comment (swatisehgal) [inline README.md]: I believe the `/pods` kubelet endpoint should include the UID information. Kubelet pods API endpoint basically encodes a list of pods. Here is a reference to that in the code: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/server/server.go#L692 and I also noticed in [this](https://www.deepnetwork.com/blog/2020/01/13/kubelet-api.html#pods) that  pointed to the [pod entry]( https://www.deepnetwork.com/blog/assets/2020-01-13-kubelet-api/pods_entry.txt).

Comment (swatisehgal) [reply README.md]: Ah, looks like you got the description swapped?

**Comment by swatisehgal** [comment]
Also, I think it is worth mentioning that currently it is not a matter of huge concern because controllers like deployments, replicasets, daemonsets etc generate the names which decreases the likelihood of conflicting pod names. However, solving this would become important once support for in place VPA is merged  in Kubernetes.

File: README.md (hunk 1 above)
Comment (ffromani) [reply README.md]: yes, fixing. Sorry.

**Comment by ffromani** [comment]
> Also, I think it is worth mentioning that currently it is not a matter of huge concern because controllers like deployments, replicasets, daemonsets etc generate the names which decreases the likelihood of conflicting pod names. However, solving this would become important once support for in place VPA is merged in Kubernetes.

indeed. Done in the last upload.

**Pull request merged**